<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdersForm</class>
 <widget class="QWidget" name="OrdersForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grocery Store</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="ProductsList">
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Products List </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QTableWidget" name="products_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="OutofstockGropuBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>449</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="view_out_of_stock_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View Out of Stock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="select_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>460</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="OrderGroupBox">
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Orders</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QTableWidget" name="order_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>301</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Order ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="order_id_box">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="view_order_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View Order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_order_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create Order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_order_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete Order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="process_order_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Process Order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_processed_orders">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View Processed Orders</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_pending_orders">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View Pending Orders</string>
      </property>
     </widget>
     <widget class="QPushButton" name="find_order_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Find Order</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="customer_name_box">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="customer_address_box">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="price_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="order_price_box">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="update_order_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update Order</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
